rules/falco_rules.yaml

```diff
@@ -905,6 +905,9 @@
 - macro: calico_writing_envvars
   condition: (proc.name=start_runit and fd.name startswith "/etc/envvars" and container.image.repository endswith "calico/node")
 
+- macro: snapd_writing_service
+  condition: (proc.name=snapd and (fd.name startswith "/etc/systemd/system/snap-amazon" or fd.name startswith "/etc/systemd/system/snap.amazon-ssm"))
```

Comment:
Maybe we should specify that this condition is amazon specific. I'm not a `snapd` expert, but I assume this macro is not just for snap.

Reply:
Yeah, you're absolutely right. What do you think about something like `snapd_installing_aws_ssm_agent`?

Reply:
Not really sure about the naming.
/cc @Kaizhe 
/cc @leodido

Reply:
can you add `snap` in the `package_mgmt_binaries` list which is already in the whitelist for writing below etc.

Reply:
Oh wow, sorry for missing that! I did it just now :smile: